homework3/homework3.py

```diff
@@ -0,0 +1,103 @@
+import csv
+import requests
+
+from faker import Faker
+from flask import Flask, jsonify
+from http import HTTPStatus
+from webargs import fields, validate
+from webargs.flaskparser import abort, use_kwargs
+
+app = Flask(__name__)
+
+fake = Faker()
+
+
+@app.errorhandler(HTTPStatus.UNPROCESSABLE_ENTITY)
+@app.errorhandler(HTTPStatus.BAD_REQUEST)
+def error_handler(error):
+    headers = error.data.get('headers', None)
+    messages = error.data.get('messages', ["Invalid request."])
+
+    if headers:
+        return jsonify(
+            {
+                'errors': messages
+            },
+            error.code,
+            headers
+        )
+    else:
+        return jsonify(
+            {
+                'errors': messages
+            },
+            error.code,
+        )
+
+
+@app.route("/students")
+@use_kwargs(
+    {
+        'count': fields.Int(
+            load_default=10,
+            validate=[validate.Range(min=1, max=1000)]
+        ),
+    },
+    location="query"
+)
+def generate_students(count):
+    with open('students.csv', 'w') as my_csv:
```

Comment:
bad idea,
try another approach:
1. Generate data and keep it in variable
2. Same data save to file 
3. Same data show on frontend